src/WebJobs.Script/Host/DefaultFunctionMetadataProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal class DefaultFunctionMetadataProvider : IFunctionMetadataProvider
+    {
+        private readonly IEnvironment _environment;
+        private readonly ILogger<DefaultFunctionMetadataProvider> _logger;
+        private IWorkerFunctionMetadataProvider _workerFunctionMetadataProvider;
+        private IHostFunctionMetadataProvider _hostFunctionMetadataProvider;
+
+        public DefaultFunctionMetadataProvider(ILogger<DefaultFunctionMetadataProvider> logger, IWorkerFunctionMetadataProvider workerFunctionMetadataProvider, IHostFunctionMetadataProvider hostFunctionMetadataProvider)
+        {
+            _logger = logger;
+            _workerFunctionMetadataProvider = workerFunctionMetadataProvider;
```

Comment:
Not null checks for `workerFunctionMetadataProvider` and `hostFunctionMetadataProvider`

Reply:
done.